### PR TITLE
RTP: rework code

### PR DESCRIPTION
**Pull request by IvanNardi**
Try avoiding false positives: look for 3 RTP packets before classifing the flow as such.

Add a generic function `is_rtp_or_rtcp()` to identify RTP/RTCP packets also in other dissectors (see 3608ab01b commit message for an example)

### CODE REVIEW

File: src/lib/protocols/rtp.c [hunk 1]
```diff
@@ -165,25 +174,83 @@ static u_int8_t isZoom(struct ndpi_flow_struct *flow,
   return(0);
 }
 
+int is_rtp_or_rtcp(struct ndpi_detection_module_struct *ndpi_struct,
```
Comment (utoni) [inline src/lib/protocols/rtp.c:183]: It makes sense to use an `enum` as retval for a better documentation.

File: src/lib/protocols/rtp.c [hunk 2]
```diff
@@ -165,25 +174,83 @@ static u_int8_t isZoom(struct ndpi_flow_struct *flow,
...
-     || (d_port == 5355 /* LLMNR_PORT */)
-     || (d_port == 5353 /* MDNS_PORT */)
-     ) {
+  if(d_port == 5355 || /* LLMNR_PORT */
+     d_port == 5353    /* MDNS_PORT */) {
     NDPI_EXCLUDE_PROTO(ndpi_struct, flow);
     return;
   }
 
+  /* TODO: should we move zoom stuff in a new, separated dissector? */
```
Comment (utoni) [inline src/lib/protocols/rtp.c:259]: Agree.

File: src/lib/protocols/rtp.c (hunk 1 above)
Comment (IvanNardi) [reply src/lib/protocols/rtp.c:183]: I agree. Done

File: src/lib/protocols/rtp.c (hunk 2 above)
Comment (IvanNardi) [reply src/lib/protocols/rtp.c:259]: I will try something in a follow-up

**Review by utoni — APPROVED** [review]

**Pull request merged**